db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_08_10_064022) do
+ActiveRecord::Schema.define(version: 2022_08_16_083515) do
```

Comment:
remove that change

Reply:
sir  2022_08_10_064022 yhi edit krna hai kya

Reply:
why its showing any new changes?

Reply:
why changes are coming in that file?